src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -203,46 +203,18 @@ public void OnRenderCompleted(long incommingBatchId, string errorMessageOrNull)
 
             // We peek first in case we are receiving a duplicate or out of order ack from
```

Comment:
Looks like the comment should be updated since this no longer peeks

Reply:
done